overlord/snapstate/booted.go

```diff
@@ -54,7 +51,7 @@ func nameAndRevnoFromSnap(sn string) (string, snap.Revision, error) {
 // still has the "active" version set to "v2" which is
 // misleading. This code will check what kernel/os booted and set
 // those versions active.
```

Comment:
Can we add here something like "To do this it creates a Change and kicks start it directly."